Source/WTF/wtf/glib/GRefPtr.h

```diff
@@ -34,8 +34,11 @@ extern "C" {
     GDBusNodeInfo* g_dbus_node_info_ref(GDBusNodeInfo*);
     void g_dbus_node_info_unref(GDBusNodeInfo*);
 };
+
+#if !PLATFORM(QT)
```

Comment:
This change certainly belongs to a different commit

Reply:
Definitely. My plan is to try to get the builds working and then extract out the work separately. I only have CI to reproduce this so need to rewrite history anyway.